user_templates/variables.cfg

```diff
@@ -171,6 +171,15 @@ variable_material_parameters: {
     }
 
 
+################################################
+## ERCF 
+################################################
+## this section is only considered if an ercf is installed and configured
+
+variable_ercf_unload_on_cancel_print: False
+variable_ercf_unload_on_end_print: True
+variable_ercf_reset_stats_on_start_print: False
```

Comment:
Check the discutions in the previsous comment by I'm quite unsure if we need all these variables

Reply:
i do believe that we need them.
Why would Moggieuk and EtteGit write statistics in a persistent file when it is erased at begin of every print?..........